script/index.js

```diff
@@ -54,56 +50,50 @@ function dynamicLoad() {
   const bookWrapper = document.createElement('div');
   bookWrapper.className = 'book_wrapper';
 
-  for (let i = 0; i < books.length; i += 1) {
+  for (let i = 0; i < awesomeBooks.books.length; i += 1) {
     const div = document.createElement('div');
     div.className = 'outputcard';
+    const p0 = document.createElement('p');
+    p0.className = 'list-books';
+    p0.innerText = `"${awesomeBooks.books[i].title}" by ${awesomeBooks.books[i].author}`;
+    div.appendChild(p0);
     const p1 = document.createElement('p');
     p1.className = `book-title_${i}`;
-    p1.innerText = books[i].title;
+    p1.innerText = awesomeBooks.books[i].title;
     div.appendChild(p1);
     const p2 = document.createElement('p');
     p2.className = `author-name_${i}`;
```

Comment:
- [ ] as the requirements you should refactor all the functions into methods' class and also object should be inside the classes so I encourage you to do that in  order that  your code will be clean and professional